lib/records/unit-tests/ink_memory_stubs.h

```diff
@@ -0,0 +1,44 @@
+/** @file
+
+  Stubs of memory allocation routines for libts.
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
```

Comment:
What is this for? Do we really need this ??

Reply:
I had assumed all new source files needed a license header. I can remove it if that is not the case.

Reply:
We don't need to stub, however we will need to link in for records almost everything else to build traffic_server. or we define stubs and link in less. OR we clean up our source layout and structure so we don't have this problem

Reply:
I have to say, I don't see anything actually useful in this file.